assets/src/dashboard/components/pageStructure/index.js

```diff
@@ -179,9 +181,12 @@ export function LeftRail() {
     >
       <div ref={upperContentRef}>
         <Content>
-          <WebStoriesHeading>
-            {__('Web Stories', 'web-stories')}
-          </WebStoriesHeading>
+          <Header>
+            <WebStoriesLogo aria-hidden={true} />
+            <WebStoriesHeading>
+              {__('Web Stories', 'web-stories')}
+            </WebStoriesHeading>
+          </Header>
```

Comment:
As per my other comment, I was wondering why we can't just use the full SVG provided in #4408.

Instead of `aria-hidden` it would need to be something like

```jsx
<WebStoriesLogo title={__('Web Stories', 'web-stories')} />
```

to make it accessible

Reply:
That's a good suggestion, I didn't realize the glyphs were converted to paths.

Reply:
Sure thing, I did it this way to keep the heading hierarchy in place. I'll just add a hidden h1 tag instead.